### PR TITLE
App test

### PR DESCRIPTION
**Pull request by sadov**
Application state detector and refactored `2dockers_test` escript included to workflow test set.

### CODE REVIEW

**Review by mydatascience — CHANGES_REQUESTED** [review]
Tests failing
http://eden-jenkins.fleandr.com:8080/job/eve-docker-image/107/console

File: tests/app/app_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+				}
+				return fmt.Errorf(out)
+			}
+		} else {
+			if msg.Ztype == info.ZInfoTypes_ZiApp {
+				for _, appName := range appNames {
+					if msg.GetAinfo().AppName == appName {
+						astate := msg.GetAinfo().State.String()
+						if state == astate {
+							return fmt.Errorf(
```
Comment (giggsoff) [inline tests/app/app_test.go:75]: We need to check all apps inside appNames, not return on the first check.

Comment (sadov) [reply tests/app/app_test.go:75]: Thanks, I will take a look.

**Comment by mydatascience** [comment]
Now it's ok

**Review by rvs — COMMENTED** [review]
LGTM

**Pull request merged**